exercises/triangle/triangle_test.py

```diff
@@ -3,55 +3,45 @@
 from triangle import Triangle, TriangleError
 
 
+# Tests adapted from `problem-specifications/exercises/\
+# triangle/canonical-data.json` @ v1.0.0
+
 class TriangleTests(unittest.TestCase):
-    def test_equilateral_triangles_have_equal_sides(self):
+    def test_equilateral_triangles(self):
         self.assertEqual(Triangle(2, 2, 2).kind(), "equilateral")
+        self.assertNotEqual(Triangle(2, 3, 2).kind(), "equilateral")
+        self.assertNotEqual(Triangle(5, 4, 6).kind(), "equilateral")
+        # self.assertNotEqual(Triangle(0, 0, 0).kind(), "equilateral")
+        self.assertEqual(Triangle(0.5, 0.5, 0.5).kind(), "equilateral")
 
-    def test_larger_equilateral_triangles_also_have_equal_sides(self):
-        self.assertEqual(Triangle(10, 10, 10).kind(), "equilateral")
-
-    def test_isosceles_triangles_have_last_two_sides_equal(self):
+    def test_isosceles_triangles(self):
         self.assertEqual(Triangle(3, 4, 4).kind(), "isosceles")
-
-    def test_isosceles_triangles_have_first_and_last_sides_equal(self):
         self.assertEqual(Triangle(4, 3, 4).kind(), "isosceles")
-
-    def test_isosceles_triangles_have_two_first_sides_equal(self):
         self.assertEqual(Triangle(4, 4, 3).kind(), "isosceles")
+        # self.assertEqual(Triangle(4, 4, 4).kind(), "isosceles")
+        self.assertNotEqual(Triangle(2, 3, 4).kind(), "isosceles")
+        # self.assertNotEqual(Triangle(1, 1, 3).kind(), "isosceles")
+        self.assertEqual(Triangle(0.5, 0.4, 0.5).kind(), "isosceles")
 
-    def test_isosceles_triangles_have_in_fact_exactly_two_sides_equal(self):
-        self.assertEqual(Triangle(10, 10, 2).kind(), "isosceles")
-
-    def test_scalene_triangles_have_no_equal_sides(self):
-        self.assertEqual(Triangle(3, 4, 5).kind(), "scalene")
-
-    def test_scalene_triangles_have_no_equal_sides_at_a_larger_scale_too(self):
-        self.assertEqual(Triangle(10, 11, 12).kind(), "scalene")
-
-        self.assertEqual(Triangle(5, 4, 2).kind(), "scalene")
+    def test_scalene_triangles(self):
+        self.assertEqual(Triangle(5, 4, 6).kind(), "scalene")
+        self.assertNotEqual(Triangle(4, 4, 4).kind(), "scalene")
+        self.assertNotEqual(Triangle(4, 4, 3).kind(), "scalene")
+        # self.assertNotEqual(Triangle(7, 3, 2).kind(), "scalene")
+        self.assertEqual(Triangle(0.5, 0.4, 0.6).kind(), "scalene")
 
-    def test_very_small_triangles_are_legal(self):
-        self.assertEqual(Triangle(0.4, 0.6, 0.3).kind(), "scalene")
+    # Additional tests for this track
 
-    def test_triangles_with_no_size_are_illegal(self):
+    def test_illegal_triangles(self):
         with self.assertRaises(TriangleError):
-            Triangle(0, 0, 0)
+            Triangle(0, 0, 0).kind()
+            Triangle(3, 4, -5).kind()
+            Triangle(1, 1, 3).kind()
+            Triangle(2, 4, 2).kind()
+            Triangle(7, 3, 2).kind()
```

Comment:
This block will only test that one of these method calls raises a `TriangleError`. In order to test all of these, each should be within its own `assertRaises` block - the previous format with separate tests would be preferable.

I also think that it would be better for invalid triangles to raise an error on creation, rather than being able to exist and only cause an error when used, so calls should be to `Triangle`, not `Triangle.kind()`.